src/crypto/symmetric_key.h

```diff
@@ -40,16 +40,16 @@ namespace crypto
       *reinterpret_cast<uint64_t*>(iv) = seq;
     }
 
-    void set_iv_id(uint64_t id)
+    void set_iv_term(uint64_t term)
```

Comment:
nit: I think this should be view

Reply:
As discussed in another comment, the KV uses `term` rather than view so that it's not confused with a `kv::Tx`'s view. Because the KV drives the encryptor, I think it is preferable to use `term` here.

Reply:
If/when we kill off usage of the "view" in "get_view" in the KV and the associated view types, we can do this rename. But until then, we're better off not having two different concepts called "views" in KV.

Another possibility, is to use consensus_view/ConsensusView for this in the KV.